eng/common/scripts/Verify-Links.ps1

```diff
@@ -115,12 +117,14 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 function CheckLink ([System.Uri]$linkUri)
 {
   if ($checkedLinks.ContainsKey($linkUri)) { return }
+  $checkedLinks[$linkUri] = $true;
```

Comment:
We should not directly access the hashtable through the this function as it might become a threading issue if we ever try to run this in parallel. Instead please use a local `$linkValid` variable instead and only read the hashtable at the top and set it at the bottom of this function.